src/async_context_frame.h

```diff
@@ -0,0 +1,33 @@
+#ifndef SRC_ASYNC_CONTEXT_FRAME_H_
+#define SRC_ASYNC_CONTEXT_FRAME_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include "base_object.h"
+#include "v8.h"
+
+#include <cstdint>
+
+namespace node {
+namespace async_context_frame {
+
+class Scope {
```

Comment:
Should we disallow copy/assignment? Or is it anyway implicit because of the members?

Reply:
Not sure. Any @nodejs/cpp-reviewers have opinions on this? I don't have strong opinions either way.